src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -358,24 +1638,16 @@ resource pipeline_extractExport 'Microsoft.DataFactory/factories/pipelines@2018-
   }
 }
 
+//------------------------------------------------------------------------------
 //------------------------------------------------------------------------------
 // Export container transform pipeline
-// Trigger: pipeline_extractExport
-//
 // Converts CSV files to Parquet or .CSV.GZ files.
 //------------------------------------------------------------------------------
-
 resource pipeline_transformExport 'Microsoft.DataFactory/factories/pipelines@2018-06-01' = {
   name: '${safeExportContainerName}_transform'
   parent: dataFactory
-  dependsOn: [
-    dataset_msexports
-    dataset_ingestion
-  ]
   properties: {
```

Comment:
Given your workflow is to work in DF Studio and copy the properties into bicep, would it be more efficient to store each of these in separate JSON files and pull them in via `loadJsonContent()`? This can be a separate PR. Just calling it out in case that would streamline the dev flow.